docs/ca/dev/preparation-for-going-live.md

```diff
@@ -84,8 +84,8 @@ Update them if plans change.
+| TES-06 | Testing | Conduct stress and performance tests                   | Test your production environment and assess its performance. Because production environments typically employ horizontal auto-scaling, it's essential to test under expected average and peak loads. These tests enable our team to optimize the environment's vertical scaling in advance, ensuring that it can seamlessly handle the expected loads from the get-go without any potential delays caused by auto-scaling mechanisms. This proactive approach eliminates the need for post-launch adjustments. For more information, see [Environment scaling](https://docs.spryker.com/docs/ca/dev/environment-scaling.html). <br> Inform us about your load and performance test plans via the [Support Portal](https://support.spryker.com/s/) at least 3 days in advance and share the results so that we can fine-tune the environment to meet your specific requirements.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | Maintain active communication with us. Inform us about your load and performance test plans and share the results so that we can fine-tune the environment to meet your  requirements.                                                                                                                                                                                                                                                                                                                                                                                                                                                   
```

Comment:
@andriitserkovnyi can we make "at least 3 days in advance" more visible as well?